float64 x
float64 y
float64 z
float64 v
float64 depth

int32 keypoint_number
string keypoint_name
